GVFS/GVFS.UnitTests/Common/TryCreateProductUpgraderTests.cs

```diff
@@ -132,6 +155,50 @@ public void NoUpgraderWhenNuGetFeedMissing()
             error.ShouldNotBeNull();
         }
 
+        [TestCase]
+        public void NoOrgUpgraderWhenNuGetPackNameMissing()
+        {
+            MockLocalGVFSConfig gvfsConfig = this.ConstructDefaultMockOrgNuGetConfigBuilder()
+                .WithNoUpgradeFeedPackageName()
+                .Build();
+
+            bool success = ProductUpgrader.TryCreateUpgrader(
+                this.tracer,
+                this.fileSystemMock.Object,
+                gvfsConfig,
+                this.credentialStoreMock.Object,
+                false,
+                false,
+                out ProductUpgrader productUpgrader,
+                out string error);
+
+            success.ShouldBeFalse();
+            productUpgrader.ShouldBeNull();
+            error.ShouldNotBeNull();

```

Comment:
Do we need to validate what the value of `error` is?

Reply:
I am not against it, but I think most of the value is captured by testing that there is an error. If I test for the exact value of the error message, it makes it harder to change the details of the message.